### ISSUE TITLE
Low contrast for code markup in docs with dark theme

### ISSUE THREAD

**Issue by thanhpd** [angular/components#15262]
#### What is the current behavior?
The documentation page with dark themes are displaying the `code` tags with an insufficient contrast ratio and are really hard to read

![firefox_qtxhoxmenm](https://user-images.githubusercontent.com/6737911/53155697-3048a080-35f0-11e9-900d-0c7f23676e7b.png)
![firefox_zd73sqcpts](https://user-images.githubusercontent.com/6737911/53155710-350d5480-35f0-11e9-8961-1e3a681928c2.png)

#### What is the expected behavior?
Increase the contrast ratio for these tags so that it's easier to read.

[Reference from WCAG](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html):
> The visual presentation of text and images of text has a contrast ratio of at least 4.5:1

#### What are the steps to reproduce?
- Enable dark theme on the documentation page
- The `code` tags can be found mostly in the Overview and API tab of each page

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular Material documentation, v. 7.3.1

**Comment by surfer19** [comment]
I made some investigation, I didn't found any purple texts but I found another 2.54 - 3.48 ratio, mostly in tags.
![](https://i.imgur.com/8svuutI.png)

**Comment by twerske** [comment]
Closed as this was fixed in https://github.com/angular/material.angular.io/pull/635 - thanks for the attentiveness to accessibility of our docs @thanhpd!
